systemtest/approvals/TestServiceSummaryMetricsAggregationOverflow.approved.json

```diff
@@ -139,7 +139,7 @@
                 "name": "metric"
             },
             "service": {
-                "environment": "dev",
+                "environment": "prod",
```

Comment:
[for reviewer] service summary metrics map iteration order is non-deterministic. This change is a result of change in underlying aggregation implementation.